aws/service.ts

```diff
@@ -618,16 +613,10 @@ function placementConstraintsForHost(host: cloud.HostProperties | undefined) {
     return undefined;
 }
 
-interface ExposedPorts {
-    [name: string]: {
-        [port: string]: ExposedPort;
-    };
-}
-
 interface ExposedPort {
-    host: aws.elasticloadbalancingv2.LoadBalancer;
-    hostPort: number;
-    hostProtocol: cloud.ContainerProtocol;
+    loadBalancer: ContainerPortLoadBalancer;
+    containerName: string;
+    containerPort: number;
 }
```

Comment:
1. by pointing at ContainerPortLoadBalancer we don't need hostProtocol since you can get it off the loadbalancer.
2. i've added containerName so that we can simplify things by just passing along a single instance of this.
3. renamed hostPort to containerPort for symmetry.